### PR TITLE
docs: added example for awscc_sagemaker_image

### PR DESCRIPTION
**Pull request by vivekthangamuthu**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

File: examples/resources/awscc_sagemaker_image/sagemaker_image.tf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+resource "awscc_sagemaker_image" "example" {
+  image_name     = "example"
+  image_role_arn = aws_iam_role.test.arn
```
Comment (quixoticmonk) [inline examples/resources/awscc_sagemaker_image/sagemaker_image.tf:3]: Thank you for contribution, @vivekthangamuthu . 
- Could you update the `image_role_arn` to be a variable or a reference of an AWSCC resource?
- And, if you can please include the tags as in other examples.

Comment (vivekthangamuthu) [reply examples/resources/awscc_sagemaker_image/sagemaker_image.tf]: @quixoticmonk Thanks i have fixed the same now.

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thank you

**Comment by quixoticmonk** [comment]
@vivekthangamuthu  You may need to run a `make docs` so that the template changes are picked up.

**Comment by vivekthangamuthu** [comment]
> @vivekthangamuthu You may need to run a `make docs` so that the template changes are picked up.

@quixoticmonk Executed the same and updated the files.

**Comment by marcosentino** [comment]
@vivekthangamuthu Thank you for the contribution!

**Pull request merged**